### ISSUE TITLE
Chip: Removable chip lost the tabindex making it unfocusable with tab

### ISSUE THREAD

**Issue by benoitkopp** [primefaces/primevue#8186]
### Describe the bug

Hello,

I'm using removable chips, and I figured out that the svg remove icon has aria-hidden="true" attribute and does not have the tabindex attribute, which makes it impossible to focus with the tab key.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [x] Other

### Reproducer

https://stackblitz.com/run?file=src%2FApp.vue

### Environment

Vue 3.5.22
PrimieVue 4.4.1
Node 22.18.0

### Vue version

3.5.22

### PrimeVue version

4.4.1

### Node version

22.18.0

### Steps to reproduce the behavior

1. Display a removable chip
2. Press on tab key
3. See that the removable icon is not focused

### Expected behavior

The remove icon should be focusable with tab key
